operations/network/management_network/apply_custom_config_1.2.md

```diff
@@ -95,29 +96,50 @@ sw-spine-002 [mlag-domain: master] # conf t
 
 All that is required to re-apply the users is to get into the global configuration mode using `conf t` and paste in the configuration that was copied from the previous step.
 
-### Aruba
+### Aruba Credentials
```

Comment:
```suggestion
### Aruba credentials
```

Reply:
This still needs to be resolved. The HPE style guide indicates that headers (except for the main page title) should use sentence casing (that is, the same way you would capitalize a sentence).